addons/mail/static/src/discuss/call/common/call_menu.xml

```diff
@@ -8,7 +8,7 @@
                     <div class="o-discuss-CallMenu-channelInfo text-truncate btn btn-group-item border-0 m-0 p-0 d-flex align-items-center o-gap-0_5 ps-1" role="button">
                         <span class="position-relative small bg-transparent">
                             <i class="me-2 fa fa-fw" t-att-class="icon" />
-                            <small class="d-flex position-absolute top-0 end-0 smaller bg-inherit">
+                            <small t-if="rtc.microphonePermission === 'granted'" class="d-flex position-absolute top-0 end-0 smaller bg-inherit">
```

Comment:
The volume-up means the conversation is in call.
This menu is necessarily for calls, but the icon is there for consistency with discuss sidebar. Recent change I made shows avatar and icon is on avatar, so it's no longer misleading when it was on microphone like before.
<img width="169" height="41" alt="Image" src="https://github.com/user-attachments/assets/a089df86-7925-4bbc-ac4a-33d61ebe61d6" />